Source/JavaScriptCore/bytecode/LinkTimeConstant.h

```diff
@@ -33,6 +33,7 @@ class CodeBlock;
 class JSGlobalObject;
 
 #define JSC_FOREACH_LINK_TIME_CONSTANTS(v) \
+    BUN_JSC_LINK_TIME_CONSTANTS(v) \
```

Comment:
☝️ This lil' macro lives in CommonIdentifiers.h and co-locates all of our BUN_JSC_ADDITIONS names